runtime/src/chplsys.c

```diff
@@ -52,9 +52,65 @@
 #include <unistd.h>
 
 
-#ifndef chplGetPageSize
-#define chplGetPageSize() sysconf(_SC_PAGE_SIZE)
+size_t chpl_getSysPageSize(void) {
+  static size_t pageSize = 0;
+
+  //
+  // No conflict management.  We assume this is set once, early, before
```

Comment:
Yes, I think I could take out those comments. In an earlier version of the code I had multiple loads/stores of the page size (basically, replace tmpPageSize with pageSize in chpl_getHeapPageSize()) and I was worried about non-atomicity in the loads and stores, probably because of recent memory model discussions! But in the current code I don't think there's a problem.